chart/templates/scheduler/scheduler-deployment.yaml

```diff
@@ -85,6 +85,9 @@ spec:
         {{- if .Values.airflowPodAnnotations }}
         {{- toYaml .Values.airflowPodAnnotations | nindent 8 }}
         {{- end }}
+        {{- if .Values.scheduler.podAnnotations }}
```

Comment:
I feel like this should just be `.Values.scheduler.annotations`. Thoughts?

Reply:
While I prefer `podAnnotations`, I absolutely do not mind one way or another, and can change them all if you'd like.

The reason I prefer `podAnnotations` are:
- I'd like to stick with helm's default chart's conventions (the example chart you'd get by doing `helm create test-chart`, and in their chart they have:
```
  template:
    metadata:
      {{- with .Values.podAnnotations }}
      annotations:
        {{- toYaml . | nindent 8 }}
      {{- end }}
```
- I think the reason helm's default chart went with podAnnotations, is to make it clear it is not eg. the annotation of the `Deployment` object, an approach with which I agree

Reply:
Interesting, I never picked up on the reasoning in the example chart, makes sense, TIL. Thanks 👍